dashboards/google-cloud-composer/cloud-composer-monitor.json

```diff
@@ -0,0 +1,389 @@
+                "plotType": "LINE",
+                "targetAxis": "Y1",
+                "timeSeriesQuery": {
+                  "apiSource": "DEFAULT_CLOUD",
+                  "timeSeriesFilter": {
+                    "aggregation": {
+                      "alignmentPeriod": "60s",
+                      "crossSeriesReducer": "REDUCE_MIN",
+                      "groupByFields": [],
+                      "perSeriesAligner": "ALIGN_MIN"
+                    },
+                    "filter": "metric.type=\"composer.googleapis.com/environment/worker/min_workers\" resource.type=\"cloud_composer_environment\" resource.label.\"environment_name\"=\"sb-data-projects-composer-load-test\" resource.label.\"location\"=\"us-central1\" ${Cluster}",
+                    "secondaryAggregation": {
+                      "alignmentPeriod": "60s",
+                      "crossSeriesReducer": "REDUCE_NONE",
+                      "perSeriesAligner": "ALIGN_NONE"
+                    }
+                  }
+                }
+              },
+              {
+                "legendTemplate": "Max workers",
+                "minAlignmentPeriod": "60s",
+                "plotType": "LINE",
+                "targetAxis": "Y1",
+                "timeSeriesQuery": {
+                  "apiSource": "DEFAULT_CLOUD",
+                  "timeSeriesFilter": {
+                    "aggregation": {
+                      "alignmentPeriod": "60s",
+                      "crossSeriesReducer": "REDUCE_MAX",
+                      "groupByFields": [],
+                      "perSeriesAligner": "ALIGN_MIN"
+                    },
+                    "filter": "metric.type=\"composer.googleapis.com/environment/worker/max_workers\" resource.type=\"cloud_composer_environment\" resource.label.\"environment_name\"=\"sb-data-projects-composer-load-test\" resource.label.\"location\"=\"us-central1\" ${Project} ${Cluster}",
+                    "secondaryAggregation": {
+                      "alignmentPeriod": "60s",
+                      "crossSeriesReducer": "REDUCE_NONE",
+                      "perSeriesAligner": "ALIGN_NONE"
+                    }
+                  }
+                }
+              }
+            ],
+            "thresholds": [],
+            "timeshiftDuration": "0s",
+            "yAxis": {
+              "label": "y1Axis",
+              "scale": "LINEAR"
+            }
+          }
+        },
+        "width": 4,
+        "xPos": 8,
+        "yPos": 1
+      },
+      {
+        "height": 3,
+        "widget": {
+          "title": "Dagbag Size",
+          "xyChart": {
+            "chartOptions": {
+              "mode": "COLOR"
+            },
+            "dataSets": [
+              {
+                "plotType": "LINE",
+                "targetAxis": "Y1",
+                "timeSeriesQuery": {
+                  "timeSeriesQueryLanguage": "fetch\n  cloud_composer_environment::composer.googleapis.com/environment/dagbag_size"
```

Comment:
Not super critical, but if you're plotting a metric without doing a lot of joins or transformations, minor preference to use "Basic" instead of "MQL" for the same result:

<img width="997" alt="Screenshot 2023-02-23 at 4 14 47 PM" src="https://user-images.githubusercontent.com/6540967/221031955-bd8ab3be-75ec-45c7-8d93-c42d35ac1dfe.png">

Reply:
Let's standardize all charts this way -- for example, I see that "Database CPU", "Running Tasks", etc are MQL but "Worker Pod Durations", "Workflow Run Durations" are Basic, when all of them can be basic for consistency.

Reply:
As a bonus, the template variable filter doesn't apply to all the charts right now. Making all of them "Basic" will let you apply the filters more consistently:

<img width="458" alt="Screenshot 2023-02-23 at 4 21 54 PM" src="https://user-images.githubusercontent.com/6540967/221033089-af57bbd3-bffb-4971-8e8b-2c6589b0b739.png">

Reply:
all resolved, except Zombie tasks killed metric. This one is not available as a Basic !